lib/src/translations.dart

```diff
@@ -0,0 +1,39 @@
+class Translations {
+  final Map<String, dynamic> _translations;
+  final Map<String, dynamic> _cachedNestedKeys;
+
+  Translations(this._translations) : this._cachedNestedKeys = {};
+  String get(String key) =>
+      (isNestedKey(key) ? getNested(key) : _translations[key]);
```

Comment:
We can simplify it with `getNested(key) ?? _translations[key]`. This reduces an additional `isNestedCached` call

Reply:
We cannot because it would break the dotted keys "nested.but.not.nested". 
We only want to go to `getNested` if the key is not nested.
Comments and tests should be added to this class.

Reply:
I see, makes sense